typespec/Applications.Core/containers.tsp

```diff
@@ -224,7 +225,7 @@ model ConnectionProperties {
   source: string;
 
   @doc("default environment variable override")
-  disableDefaultEnvVars?: boolean = false;
+  disableDefaultEnvVars?: boolean;
```

Comment:
Very nice. The autoformat did this right?

Reply:
I did this. The autoformat doesn't handle this case.

Reply:
I think it's OK to do cleanup like this if we want to. We should be careful introducing rules that aren't enforced by tooling. Over time it turns into extra complexity for us. 

It might be a good idea to log an issue on typespec to suggest automatically removing `= false` from boolean values.

Reply:
Created this issue: https://github.com/microsoft/typespec/issues/3565